### ISSUE TITLE
Support Hololens2 UWP ARM64.

### ISSUE THREAD

**Issue by Xbathy** [alphacep/vosk-api#959]
Can vosk support UWP arm64 paltform, use on Hololens2 device.

**Comment by andreclerigo** [comment]
Yes, this would be a nice feature to have, as AR headsets can be seamlessly integrated with Voice Assistants
